source/existing/09_vanderhoorn.rst

```diff
@@ -1,6 +1,6 @@
-.. _Gijs Vanderhoorn:
+.. _Gijs van der Hoorn:
```

Comment:
Not sure this is correct Sphinx syntax.

Reply:
Yes, this is fine, but we actually have a reference to that label inside proposal/index.rst which needs to be altered, as well. I seem to be too stupid to link individual lines from the RST document, though...